core/src/main/java/io/cucumber/core/plugin/ProgressFormatter.java

```diff
@@ -39,7 +39,7 @@ public final class ProgressFormatter implements ConcurrentEventListener, ColorAw
     private boolean monochrome = false;
 
     public ProgressFormatter(OutputStream out) {
-        this.out = new NiceAppendable(new UTF8OutputStreamWriter(out));
+        this.out = new NiceAppendable(new UTF8OutputStreamWriter(out), true);
```

Comment:
A comment to explain why every write is flushed here would be useful.

Reply:
addressed in commit https://github.com/cucumber/cucumber-jvm/commit/4e020ded54179322aaf1b0974359258135321245.